frontend/src/components/pages/CampOverview/RemoveCamperModal/index.tsx

```diff
@@ -74,7 +74,8 @@ const RemoveCamperModal = ({
       setRetrievedCampers([]);
       setCampersToBeDeleted(new Set<Camper>());
     };
-  }, [camper, toast]);
+    // eslint-disable-next-line
+  }, [camper]);
```

Comment:
Does it cause a problem if we keep toast in the dependency array? If not I suggest we keep it here.

Reply:
Yeah, my mistake. Will put toast into dependency array.